src/build/experimental/commands/compile/wasm_binary/rust/experimental_canister_template/src/lib.rs

```diff
@@ -95,8 +90,3 @@ pub fn _azle_get_benchmarks() -> Vec<benchmarking::BenchmarkEntry> {
     benchmarking::BENCHMARKS_REF_CELL
         .with(|benchmarks_ref_cell| benchmarks_ref_cell.borrow().clone())
 }
-
-#[ic_cdk_macros::query]
```

Comment:
What this function only around to fix the memory leak? If we have another memory leak we will add it, or something similar back? Just curious